lib/WiiExtension/WiiExtension.h

```diff
@@ -145,59 +114,17 @@ class WiiExtension {
 	void reset();
   	void start();
   	void poll();
+
+    ExtensionBase* getController() { return extensionController; };
   private:
-	
+	ExtensionBase *extensionController = NULL;
+
     uint8_t iSDA;
     uint8_t iSCL;
-    uint8_t bWire;
     i2c_inst_t *picoI2C;
-
 	int32_t iSpeed;
 
-    uint16_t _minX1 = 0;
-    uint16_t _maxX1 = 1;
-    uint16_t _cenX1 = 0;
-
-    uint16_t _minY1 = 0;
-    uint16_t _maxY1 = 1;
-    uint16_t _cenY1 = 0;
-
-    uint16_t _minX2 = 0;
-    uint16_t _maxX2 = 1;
-    uint16_t _cenX2 = 0;
-
-    uint16_t _minY2 = 0;
-    uint16_t _maxY2 = 1;
-    uint16_t _cenY2 = 0;
-
-    uint16_t _accelX0G = 0;
-    uint16_t _accelY0G = 0;
-    uint16_t _accelZ0G = 0;
-    uint16_t _accelX1G = 0;
-    uint16_t _accelY1G = 0;
-    uint16_t _accelZ1G = 0;
-
-    //uint8_t _lastRead[16];
-
-    uint16_t _calibrationPrecision1From = WII_ANALOG_PRECISION_0;
-    uint16_t _calibrationPrecision1To = WII_ANALOG_PRECISION_0;
-    uint16_t _calibrationPrecision2From = WII_ANALOG_PRECISION_0;
-    uint16_t _calibrationPrecision2To = WII_ANALOG_PRECISION_0;
-
-    uint16_t _analogPrecision1From = WII_ANALOG_PRECISION_0;
-    uint16_t _analogPrecision1To = WII_ANALOG_PRECISION_0;
-    uint16_t _analogPrecision2From = WII_ANALOG_PRECISION_0;
-    uint16_t _analogPrecision2To = WII_ANALOG_PRECISION_0;
-
-    uint16_t _triggerPrecision1From = WII_ANALOG_PRECISION_0;
-    uint16_t _triggerPrecision1To = WII_ANALOG_PRECISION_0;
-    uint16_t _triggerPrecision2From = WII_ANALOG_PRECISION_0;
-    uint16_t _triggerPrecision2To = WII_ANALOG_PRECISION_0;
-
-    uint8_t _guitarType   = WII_GUITAR_UNSET;
-
-    uint16_t map(uint16_t x, uint16_t in_min, uint16_t in_max, uint16_t out_min, uint16_t out_max);
-    uint16_t calibrate(uint16_t pos, uint16_t min, uint16_t max, uint16_t center);
+    uint8_t _lastRead[16] = {0xFF};
```

Comment:
Looks like we could get rid of this since its commented out in all of your debugs?

Reply:
Will put this behind WII_EXTENSION_DEBUG